app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
Add doc strings for every error class (same for every function and class). Also, why do you use `BaseException` as a parent for this error class and `Exception` for the next one?